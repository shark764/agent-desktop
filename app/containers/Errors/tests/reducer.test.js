import errorsReducer from '../reducer';

describe('Errors Reducer', () => {
  it('returns the initial state', () => {
    expect(errorsReducer(undefined, {})).toMatchSnapshot();
  });
});
