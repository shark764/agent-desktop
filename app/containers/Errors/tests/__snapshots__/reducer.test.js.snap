// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Reducer returns the initial state 1`] = `
Immutable.Map {
  "stats": Immutable.Map {
    "erroredIds": Immutable.List [],
  },
}
`;
